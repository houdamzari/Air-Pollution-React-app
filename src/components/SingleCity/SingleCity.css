.pill {
    background-color: brown;
    color: white;
    padding: 10px 20px;
    border-radius: 5px;
    font-size: 20px;
    font-weight: bold;
    cursor: pointer;
    transition: all .4s;
}

.pill:hover {
    transform: scale(1.05);
}