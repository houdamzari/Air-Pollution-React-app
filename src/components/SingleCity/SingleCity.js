import "./SingleCity.css";
function SingleCity({ children, handleClick }) {
  return (
    <div className="pill" onClick={handleClick}>
      {children}
    </div>
  );
}

export default SingleCity;
