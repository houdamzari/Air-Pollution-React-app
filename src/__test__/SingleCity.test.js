import "@testing-library/jest-dom";
import SingleCity from "../components/SingleCity/SingleCity";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

describe("Test SingleCity Component", () => {
  test("SingleCity component should match with snapshot", () => {
    const singleCity = renderer
      .create(
        <BrowserRouter>
          <SingleCity />
        </BrowserRouter>
      )
      .toJSON();

    expect(singleCity).toMatchSnapshot();
  });
});
