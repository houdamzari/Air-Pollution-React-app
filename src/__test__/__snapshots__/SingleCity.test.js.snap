// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SingleCity Component SingleCity component should match with snapshot 1`] = `
<div
  className="pill"
/>
`;
